src/forms/TextInput.jsx

```diff
@@ -1,6 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
 
 /**
  * @module TextInput
```

Comment:
Can you give me an example? Is it just removing state?

Or can this wait... when we implement validation we will most likely be changing this component and any state, right?